src/components/wrapper/index.js

```diff
@@ -15,7 +15,7 @@ import './wrapper.css';
  */
 export const Wrapper = ({ loading, theme }) => (
   <div
-    className={classnames('kedro-pipeline', {
+    className={classnames('kedro-pipeline kedro', {
```

Comment:
Now that we have assigned the `kedro` class here, perhaps we could clean up the `kedro` class that was stated in the child components of `wrapper`, such as within the `metadata`, `LargePipelineWarning`, `Flowchart`, etc?

Reply:
I considered it, but decided to leave them: There's some value to having these classes as the immediate parent of the text elements, in addition to the top-level parent: Doing so innoculates them against unrelated font-size changes in between. Consider this case:

```html
<div class="kedro-pipeline kedro">
  <div class="sidebar">
    <div class="sidebar__row kedro">
      <p class="sidebar__row__text">Text</p>
    </div>
  </div>
</div>
```
Let's assume that the font-size of `.sidebar__row__text` is set to `1.4em`, so it's 14px:
```css
.kedro {
  font-size: 10px;
}
.sidebar__row__text {
  font-size: 1.4em;
}
```

Now look: Some foolish person has applied a font-size to `.sidebar` in `em`:
```css
.sidebar {
  font-size: 1.2em;
}
```

This should multiply the `<p>` tag's font size by 1.2, but because the `.kedro` class is applied to `.sidebar__row__text`, it resets the font-size so the parent change won't affect it.

It's perhaps overly cautious, but I think it's worth leaving in just in case, and there's little benefit to removing it. Does that make sense?

Reply:
sounds good - no harm in keeping it in the child components in that case 👍